### PR TITLE
more functions

### PR DESCRIPTION
**Pull request by TpdeaX**

### CODE REVIEW

File: bindings/2.204/GeometryDash.bro [hunk 1]
```diff
@@ -691,29 +697,30 @@ class CCAnimateFrameCache : cocos2d::CCObject {
...
+	virtual void ccTouchMoved(cocos2d::CCTouch*, cocos2d::CCEvent*);
+	virtual void ccTouchEnded(cocos2d::CCTouch*, cocos2d::CCEvent*);
+	virtual void ccTouchCancelled(cocos2d::CCTouch*, cocos2d::CCEvent*);
+	virtual void registerWithTouchDispatcher() = win 0x22fc0;
+	virtual void keyBackClicked();
+	virtual void customSetup();
+	virtual TodoReturn enterLayer() = win 0x22fa0;
 	virtual TodoReturn exitLayer() = win 0x23010;
-	virtual void showLayer(bool) = win 0x23040;
+	virtual TodoReturn showLayer(bool) = win 0x23040;
```
Comment (Cvolton) [inline bindings/2.204/GeometryDash.bro:717]: why is this changed to TodoReturn

File: bindings/2.204/GeometryDash.bro [hunk 2]
```diff
@@ -6079,8 +6107,8 @@ class GJBaseGameLayer : cocos2d::CCLayer, TriggerEffectDelegate {
 	TodoReturn toggleGroup(int, bool);
 	TodoReturn toggleLockPlayer(bool, bool);
 	TodoReturn togglePlayerStreakBlend(bool);
-	void togglePlayerVisibility(bool, bool) = win 0x1B9320;
-	TodoReturn togglePlayerVisibility(bool); //inlined on windows
+	TodoReturn togglePlayerVisibility(bool, bool);
```
Comment (Cvolton) [inline bindings/2.204/GeometryDash.bro:6110]: why is this changed to TodoReturn and the address removed

File: bindings/2.204/GeometryDash.bro [hunk 3]
```diff
@@ -10384,9 +10412,9 @@ class MusicDownloadManager : cocos2d::CCNode, PlatformDownloadDelegate {
 	TodoReturn getAllMusicTags(OptionsObjectDelegate*);
 	TodoReturn getAllSFXObjects(bool);
 	TodoReturn getAllSongs();
-	void getCustomContentURL() = win 0x2849c0;
-	cocos2d::CCObject* getDLObject(char const*);
-	TodoReturn getDownloadedSongs();
+	TodoReturn getCustomContentURL() = win 0x2849c0;
+	TodoReturn getDLObject(char const*);
```
Comment (Cvolton) [inline bindings/2.204/GeometryDash.bro:10416]: why is this changed to TodoReturn

File: bindings/2.204/GeometryDash.bro (hunk 1 above)
Comment (altalk23) [reply bindings/2.204/GeometryDash.bro:717]: probably an merge artifact

File: bindings/2.204/GeometryDash.bro [hunk 4]
```diff
@@ -691,29 +697,30 @@ class CCAnimateFrameCache : cocos2d::CCObject {
...
-	virtual TodoReturn layerVisible();
-	virtual void layerHidden() = win 0x23070;
-	virtual void enterAnimFinished() {}
-	virtual void disableUI() {}
-	virtual void enableUI() {}
+	virtual TodoReturn layerVisible() = win 0x23060;
+	virtual TodoReturn layerHidden() = win 0x23070;
+	virtual TodoReturn enterAnimFinished();
+	virtual TodoReturn disableUI();
+	virtual TodoReturn enableUI();
```
Comment (altalk23) [inline bindings/2.204/GeometryDash.bro:723]: same here probably

File: bindings/2.204/GeometryDash.bro (hunk 1 above)
Comment (TpdeaX) [reply bindings/2.204/GeometryDash.bro:717]: I didn't really notice that, too bad for me 💀

File: bindings/2.204/GeometryDash.bro (hunk 2 above)
Comment (TpdeaX) [reply bindings/2.204/GeometryDash.bro:6110]: There I saw that the parameters for precisely that address come out with only one, and Ghidra shows it as one too. I thought they made a mistake. (about the TodoReturn, I'll fix it :c)

File: bindings/2.204/GeometryDash.bro (hunk 3 above)
Comment (TpdeaX) [reply bindings/2.204/GeometryDash.bro:10416]: :c

File: bindings/2.204/GeometryDash.bro (hunk 4 above)
Comment (TpdeaX) [reply bindings/2.204/GeometryDash.bro:723]: :c
